src/.eslintrc

```diff
@@ -6,5 +6,8 @@
   },
   "plugins": [
     "babel"
-  ]
+  ],
+  "globals": {
```

Comment:
No, we've decided to use pinkie promise, so enabling `es6` will not report about `Promise` global